Dockerfile

```diff
@@ -1,12 +1,28 @@
-FROM        registry.access.redhat.com/ubi8/nodejs-14
+FROM registry.access.redhat.com/ubi8/nodejs-16
 
-RUN         npm install -g yarn
+USER 0
 
-COPY        . .
+RUN set -eux && \
+  npm set progress=false && \
+  npm set update-notifier=false && \
+  npm set audit=false && \
+  npm set fund=false && \
+  npm install -g npm@latest
 
-RUN         yarn install --production
+USER 1001
 
-EXPOSE      4242
+COPY --chown=1001:root package.json package-lock.json ./
 
-ENTRYPOINT  ["node"]
-CMD         ["index.js"]
+ENV NODE_ENV production
+
+RUN set -eux && \
+  npm install-clean && \
```

Comment:
usually `npm ci` is used directly instead of alias, https://docs.npmjs.com/cli/v9/commands/npm-ci, same result

Reply:
Done. Changed in https://github.com/app-sre/unleash/pull/42/commits/0ce1e0279ed266dd6d7839f8f8e9badaf7d7403a.

Thank you.